client/rest-high-level/src/main/java/org/elasticsearch/client/RequestConverters.java

```diff
@@ -108,6 +109,17 @@ private RequestConverters() {
         // Contains only status utility methods
     }
 
+    static Request cancelTasks(CancelTasksRequest cancelTasksRequest) {
+        Request request = new Request(HttpPost.METHOD_NAME, "/_tasks/_cancel");
+        Params params = new Params(request);
+        params.withTimeout(
+            cancelTasksRequest.getTimeout())
+            .withTaskId(cancelTasksRequest.getTaskId())
+            .withNodes(cancelTasksRequest.getNodes())
+            .withActions(cancelTasksRequest.getActions());
```

Comment:
the REST action seems to also support parent_task_id, the parameter is also declared in the REST spec so it should be here too?